<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StateMachineEditorGraphvizSettings</class>
 <widget class="QWidget" name="StateMachineEditorGraphvizSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>507</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QGroupBox" name="LayoutBox">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QGroupBox" name="AlgorithmBox">
        <property name="title">
         <string>Layout Algorithm</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QRadioButton" name="A3">
           <property name="text">
            <string>Fdp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="A1">
           <property name="text">
            <string>Dot</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="SplineBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="title">
         <string>Spline Layout</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QRadioButton" name="S1">
           <property name="text">
            <string>none</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="S2">
           <property name="text">
            <string>line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="S3">
           <property name="text">
            <string>spline</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="S4">
           <property name="text">
            <string>ortho</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="S5">
           <property name="text">
            <string>polyline</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="S6">
           <property name="text">
            <string>curved</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="S7">
           <property name="text">
            <string>straight</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="AutoZoom">
        <property name="title">
         <string>AutoZoom</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item row="1" column="1">
          <widget class="QRadioButton" name="zoom_height">
           <property name="text">
            <string>Zoom to full graph height</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="zoom_full">
           <property name="text">
            <string>Zoom to full graph size</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="zoom_width">
           <property name="text">
            <string>Zoom to full graph width</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="zoom_disabled">
           <property name="text">
            <string>Disabled</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Preview">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="PreviewLayout"/>
      </item>
      <item>
       <widget class="QPushButton" name="refresh">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
